t/t0002-basic.t

```diff
@@ -99,8 +99,8 @@ test_expect_success 'pmix.job.napps is set to 1' '
 		${GETKEY} --proc=* pmix.job.napps >pmix.job.napps.out &&
 	test_cmp pmix.job.napps.exp pmix.job.napps.out
 '
-test_expect_success 'pmix.nsdir is set' '
-	run_timeout 30 flux mini run -n1 \
+test_expect_success 'pmix.nsdir is NOT set' '
+	test_must_fail flux mini run \
```

Comment:
I wonder if, instead of testing that `flux mini run` of `getkey` fails, it would be better to add an option to `getkey` to ensure that a key is not set.

The only reason I suggest that is that there are a lot of reasons `flux mini run getkey` could fail (and thus the test could succeed) besides the requested key not being set.

Since that result is unlikely, it is fine to ignore this suggestion for now.

Reply:
Actually since it probably won't be a regular thing to test for keys not existing, I just went with this for this test:
```expecting success: 
	test_must_fail flux mini run \
		-ouserrc=$(pwd)/rc.lua \
		${GETKEY} --proc=* pmix.nsdir 2>pmix.nsdir.err &&
	grep NOT-FOUND pmix.nsdir.err

fknbnjh.0: PMIx_Get pmix.nsdir: NOT-FOUND
```
which I think accomplishes the same thing.  Sound ok?

Reply:
Yeah! works for me